README.md

```diff
@@ -161,6 +162,19 @@ make sure that there is no error forwarding the telemetry data.
 Finally, there also the [troubleshooting guide](https://github.com/open-telemetry/opentelemetry-java-instrumentation#troubleshooting)
 of the upstream OpenTelemetry Instrumentation for Java.
 
+
+## Community
+
+To engage with the Grafana Application Observability community:
+
+* Chat with us on our community Slack channel. To invite yourself to the
+  Grafana Slack, visit [https://slack.grafana.com/](https://slack.grafana.com)
+  and join the [#application-observability](https://grafana.slack.com/archives/C05E87XRK3J)
+channel.
+* Ask questions on the [Discussions page](https://github.com/grafana/grafana-opentelemetry-java/discussions).
```

Comment:
You should enable discussions in the repo settings to make this work.

And maybe disable the Wiki, as we'll likely not use it.

Reply:
thanks